src/imgtools/betapipeline.py

```
@@ -0,0 +1,97 @@
+    # nnunet_inference=False,
+    # dataset_json_path="",
+    # continue_processing=False,
+    # dry_run=False,
+    # verbose=False,
+    # update=False,
+    # roi_select_first=False,
+    # roi_separate=False
+
+    def __post_init__(self):
+        self.crawl = Crawler()
+        self.lacer = Interlacer()
```

Ruff (0.8.2)

Undefined name `Interlacer`

(F821)